doc/source/serve/production.md

```diff
@@ -583,3 +565,218 @@ This section summarizes the best practices when deploying to production:
+```console
+$ kubectl create -k "github.com/ray-project/kuberay/manifests/cluster-scope-resources?ref=v0.3.0"
+$ kubectl apply -k "github.com/ray-project/kuberay/manifests/base?ref=v0.3.0"
+$ kubectl get deployments -n ray-system
+NAME           READY   UP-TO-DATE   AVAILABLE   AGE
+ray-operator   1/1     1            1           40s
+
+$ kubectl get pods -n ray-system
+NAME                            READY   STATUS    RESTARTS   AGE
+ray-operator-75dbbf8587-5lrvn   1/1     Running   0          31s
+```
+
+For more details, see the [KubeRay installation guide](https://ray-project.github.io/kuberay/deploy/installation/).
+
+### Deploying a Serve application
+
+Once the KubeRay controller is running, you can manage your Ray Serve application by creating and updating a `RayService` CR.
+`RayService` custom resources consist of the following:
+- a `KubeRay` `RayCluster` config defining the parameters for the cluster that the Serve application runs on.
+- a Ray Serve [config](serve-in-production-config-file) defining the Serve application to run on the cluster.
+
+When the `RayService` is created, the `KubeRay` controller first creates a Ray cluster using the provided configuration.
+Then, once the cluster is running, it deploys the Serve application to the cluster using the [REST API](serve-in-production-deploying).
+The controller also creates a Kubernetes Service that can be used to route traffic to the Serve application.
+
+Let's see this in action by deploying the same [`FruitStand` example](serve-in-production-example) from above.
+The Serve config for the example is embedded into [this example `RayService` CR](https://github.com/ray-project/kuberay/blob/release-0.3/ray-operator/config/samples/ray_v1alpha1_rayservice.yaml).
+To follow along, save this CR locally in a file named `ray_v1alpha1_rayservice.yaml`.
+
+To deploy the example, we simply `kubectl apply` the CR.
+This creates the underlying Ray cluster, consisting of a head and worker node pod (see {ref}`_cluster-key-concepts-under-construction` for more details on Ray clusters), as well as the service that can be used to query our application:
+
+```console
+$ kubectl apply -f ray_v1alpha1_rayservice.yaml
+
+$ kubectl get rayservices
+NAME                AGE
+rayservice-sample   7s
+
+$ kubectl get pods
+NAME                                                      READY   STATUS    RESTARTS   AGE
+ervice-sample-raycluster-qd2vl-worker-small-group-bxpp6   1/1     Running   0          24m
+rayservice-sample-raycluster-qd2vl-head-45hj4             1/1     Running   0          24m
+
+$ kubectl get services
+NAME                                               TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                                          AGE
+kubernetes                                         ClusterIP   10.100.0.1       <none>        443/TCP                                          62d
+# A head node service maintained by the RayService.
```

Comment:
Not sure that the "maintained by the XXX" parts are important here. Cut?

Reply:
clarified that the first three are:
`# Services used internally by the KubeRay controller.`